projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -4872,5 +4872,77 @@ public string FatherEducationLevelEditFlagHelper
             get => GetObservationValueHelper();
             set => SetObservationValueHelper(value, VR.ValueSets.EditBypass01234.Codes);
         }
+
+        private int? GetIntegerObservationValue(string code)
```

Comment:
The helper functions I created could be used here, but would need to wait for that branch to get merged in.

Reply:
Done.